### PR TITLE
Myriad 198

### PR DESCRIPTION
**Pull request by hokiegeek2**
Code changes pertaining to JIRA ticket MYRIAD-198

### CODE REVIEW

File: myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java [hunk 1]
```diff
@@ -196,23 +206,23 @@ public String getMesosMaster() {
...
 
   public Optional<MyriadContainerConfiguration> getContainerInfo() {
     return Optional.fromNullable(containerInfo);
   }
 
-  public Double getFrameworkFailoverTimeout() {
-    return this.frameworkFailoverTimeout != null ? this.frameworkFailoverTimeout : DEFAULT_FRAMEWORK_FAILOVER_TIMEOUT_MS;
+  public String getFrameworkRole() {
+    return Optional.fromNullable(frameworkRole).or("*");
   }
```
Comment (DarinJ) [inline myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java]: should use DEFAULT_ROLE defined above.

**Comment by DarinJ** [comment]
@hokiegeek2 one minor nit
@smarella @sarjeet2013 @yufeldman any comments?  Otherwise I'd like to merge as this is some nice clean up.

**Comment by hokiegeek2** [comment]
Good catch, fixed, and pushed back up to git.

--John

**Comment by sarjeet2013** [comment]
LGTM.. 

Minor nit: There are few references left for 'nodeManagerUri' which needs to fixed as well. For some reason, I can't comment on the diff :(

**Comment by hokiegeek2** [comment]
Hi Sarjeet,

No worries, I'll find 'em and fix 'em.

--John

**Comment by hokiegeek2** [comment]
Hi Sarjeet,

I diff'd MYRIAD-198 and master and switched nodemanagerUri back to
nodeManagerUri in the four yml files.

--John

**Comment by sarjeet2013** [comment]
thanks for taking care of this. Ship it :shipit:

**Comment by hokiegeek2** [comment]
Cool, thanks Sarjeet!

--John

**Comment by DarinJ** [comment]
Merged!

**Comment by hokiegeek2** [comment]
Awesome, thanks!!

--John

**Pull request closed without merging**